packages/node/src/methods/app-instance-operations.ts

```diff
@@ -1,7 +1,22 @@
 import { AppInstance, StateChannel } from "@counterfactual/machine";
-import { AppInstanceInfo, AppState, Node } from "@counterfactual/types";
+import {
+  AppAction,
+  AppInstanceInfo,
+  AppState,
```

Comment:
Probably `AppState` and `AppAction` should be one type and they should be `SolidityStructCompatibleJSObject` which is what `AppState` currently actually represents.

https://github.com/counterfactual/monorepo/blob/master/packages/machine/src/protocol-types-tbd.ts#L8-L15

Reply:
One thing that might need to be fixed in the future is that JSON has no native numerical type suitable for `uint256`

Reply:
^ 

might have to stringify then

Reply:
@cf19drofxots we discussed earlier, best to store as `{ _hex: "0x..." }`

Reply:
Right, you'd have to encode the value in the string representation of a hex